client/src/components/GraphOverlay/GraphOverlay.js

```diff
@@ -2,32 +2,32 @@ import { Fragment } from "react";
 import { connect } from "react-redux";
 import MediaQuery from "react-responsive";
 
-import "./RotateLandscape.scss";
+import "./GraphOverlay.scss";
 import { IconRotatePhone, IconExpandWindowWidth } from "../../icons/icons.js";
 
-import { rotate_landscape_off } from "../../InfoBase/AppState.js";
+import { hide_graph_overlay } from "../../InfoBase/AppState.js";
 import { create_text_maker_component } from "../misc_util_components";
 
-import text from "./RotateLandscape.yaml";
+import text from "./GraphOverlay.yaml";
 
 const { TM, text_maker } = create_text_maker_component(text);
 
 class LandscapeOverlay extends React.Component {
```

Comment:
Kinda weird to see that everything changed to "GraphOverlay" and this didn't change. Not that it matters whole lot